jupyterhub/files/hub/z2jh.py

```diff
@@ -9,25 +9,42 @@
 
 import yaml
 
-
 # memoize so we only load config once
 @lru_cache()
 def _load_config():
-    """Load configuration from disk
+    """Load Helm chart configuration passed through a mounted ConfigMap."""
```

Comment:
Isn't this a secret now?

Reply:
@minrk we have a configmap for passing files, such as z2jh.py and jupyterhub_config.py, which no longer passes any values.yaml content.